src/core/Akka/Dispatch/SysMsg/ISystemMessage.cs

```diff
@@ -261,12 +261,14 @@ public ActorTaskSchedulerMessage(ActorTaskScheduler scheduler, Task task)
         ///     Initializes a new instance of the <see cref="ActorTaskSchedulerMessage" /> class.
         /// </summary>
         /// <param name="exception">The exception.</param>
-        public ActorTaskSchedulerMessage(Exception exception)
+        public ActorTaskSchedulerMessage(Exception exception,object message)
         {
             Exception = exception;
+            Message = message;
         }
```

Comment:
`ActorTaskSchedulerMessage`  now also carries the user message that caused the exception.